spec/models/preserved_copy_spec.rb

```diff
@@ -180,4 +180,20 @@
       expect(pc.changed?).to eq true
     end
   end
+
+  context '#matches_po_current_version?' do
+    it 'returns true when its version matches its preserved objects current version' do
+      preserved_copy.version = 1
+      preserved_object.current_version = 1
```

Comment:
so the code is actually comparing `preserved_copy.preserved_object`'s version, not the plain `preserved_object` that is floating around in the spec.  I think this test could read:

```ruby
it 'returns true when its version matches its preserved objects current version' do
      preserved_copy.version = 1234
      preserved_copy.preserved_object.current_version = 1234
      expect(preserved_copy.matches_po_current_version?).to be true
    end
```

note that I picked version numbers that are highly unlikely to be occurring in our fixture objects ...

Reply:
Note that this PASSES:

```ruby
    it 'returns true when its version matches its preserved objects current version' do
      preserved_copy.version = 1
      preserved_object.current_version = 8
      expect(preserved_copy.matches_po_current_version?).to be true
    end
```